src/Plugin.Fingerprint/Abstractions/AuthenticationRequestConfiguration.cs

```diff
@@ -60,6 +69,13 @@ public AuthenticationRequestConfiguration(string title, string reason)
         }
     }
 
+#if NET6_0_ANDROID
+    [SupportedOSPlatform("android")]
+#elif NET6_0_IOS
+    [SupportedOSPlatform("ios")]
+#elif NET6_0_MACCATALYST
+    [SupportedOSPlatform("maccatalyst")]
+#endif
```

Comment:
Not sure what is the purpose of this annotation, it doesn't like it is referencing any platform specific code.  Are you seeing a warning in this class?